lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -41,7 +40,7 @@ class UsesOptimizedImages extends Audit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Has optimized images',
+      description: 'Avoids unoptimized images',
```

Comment:
Affirmative descriptions still feel better to me (where we can use them). How about aligning with PSI? They use  a singular "Optimize images".  WDYT about "Optimizes images" for us?